rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -3644,24 +3647,32 @@ static bool ppu_store_reservation(ppu_thread& ppu, u32 addr, u64 reg_value)
 				{
 				case 0:
 				{
-					// Nothing to do
+					// Instead of skipping the notification, postpone the notification; giving a chance the PPU
```

Comment:
You are masking a bug.

Reply:
I simply restored, on the current logic, the step that was removed by the original regressing PR.
But obviously, only you can have a clear picture of any possible use cases.
As usual, it is not a problem for me to close the PR in case you will take care of it. Let me know